app/exceptions.py

```diff
@@ -11,3 +11,9 @@ class InvalidToken(Exception):
     """
     pass
 
+
+class UserAlreadyExist(Exception):
+    """
+    This Will be raised incases where a user already exist in the db 
+    """
+    pass
```

Comment:
consider removing code that is not doing any data storage or manipulation

Reply:
well noted @EinsteinCarrey